web/app/models.py

```diff
@@ -226,5 +223,30 @@ def __str__(self):
             self.director,
             self.rule_set,
             self.submitted)
+
+class TournamentPlayer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(80))
+    aga_num = db.Column(db.String(5))
+    member_ex_date = db.Column(db.String(20))
+    rating = db.Column(db.String(5))
+    affiliation = db.Column(db.String(80))
+    state = db.Column(db.String(2))
+    address = db.Column(db.String(80))
+    email = db.Column(db.String(80))
+    phone = db.Column(db.String(20))
+    citizenship = db.Column(db.String(80))
+    dob = db.Column(db.String(20))
+    tournament_id = db.Column(db.Integer, db.ForeignKey('tournament.id'))
+    tournament = relationship(Tournament,
+        backref=db.backref('Tournament', cascade='all, delete-orphan'))
```

Comment:
Hm. I could be wrong, but I don't think that's necessary. One can always perform a SQL join to get data from the relevant tournament. It certainly makes in much simpler not to have it.